.github/CODEOWNERS

```diff
@@ -875,9 +875,9 @@ packages/kbn-test-eui-helpers @elastic/kibana-visualizations
 x-pack/test/licensing_plugin/plugins/test_feature_usage @elastic/kibana-security
 packages/kbn-test-jest-helpers @elastic/kibana-operations @elastic/appex-qa
 packages/kbn-test-subj-selector @elastic/kibana-operations @elastic/appex-qa
-x-pack/test_serverless @elastic/appex-qa
-test @elastic/appex-qa
-x-pack/test @elastic/appex-qa
+x-pack/test_serverless 
+test 
+x-pack/test 
```

Comment:
automatically added by CI, I guess it is fine as long no one is assigned? 
@afharo wdyt?